lib/surgex/repo_helpers/repo_helpers.ex

```diff
@@ -63,4 +67,17 @@ defmodule Surgex.RepoHelpers do
       opts
     end
   end
+  @doc """
```

Comment:
let's add new line character above and we're good to go!

Reply:
Actually, we could also consider adding tests (I thought there's no test suite, but there is)